build error - 1.6.5

During build time, the following error message is displayed and build failed. 'Apache Ant(TM) version 1.9.4 compiled on April 29 2014' is used on Cygwin.

error message
============
```
untar-unzip-checksum:
     [echo] Testing checksum of "Firmata-2.4.3.zip"

unzip:
   [rename] DEPRECATED - The rename task is deprecated.  Use move instead.

BUILD FAILED
```

--- a/build/build.xml
+++ b/build/build.xml
@@ -187,14 +187,14 @@
       <param name="final_folder" value="${target.path}/libraries/Firmata-2.4.3" />
       <param name="dest_folder" value="${target.path}/libraries" />
     </antcall>
-    <rename src="${target.path}/libraries/Firmata-2.4.3" dest="${target.path}/libraries/Firmata"/>
+	<move file="${target.path}/libraries/Firmata-2.4.3" tofile="${target.path}/libraries/Firmata" />
     <antcall target="unzip">
       <param name="archive_file" value="Temboo-1.1.2.zip" />
       <param name="archive_url" value="https://github.com/temboo/Temboo/archive/1.1.2.zip" />
       <param name="final_folder" value="${target.path}/libraries/Temboo-1.1.2" />
       <param name="dest_folder" value="${target.path}/libraries" />
     </antcall>
-    <rename src="${target.path}/libraries/Temboo-1.1.2" dest="${target.path}/libraries/Temboo"/>
+	<move file="${target.path}/libraries/Temboo-1.1.2" tofile="${target.path}/libraries/Temboo" />
   </target>
 
   <!-- copy hardware folder -->

--- a/build/build.xml
+++ b/build/build.xml
@@ -187,14 +187,14 @@
       <param name="final_folder" value="${target.path}/libraries/Firmata-2.4.3" />
       <param name="dest_folder" value="${target.path}/libraries" />
     </antcall>
-	<move file="${target.path}/libraries/Firmata-2.4.3" tofile="${target.path}/libraries/Firmata" />
+    <move file="${target.path}/libraries/Firmata-2.4.3" tofile="${target.path}/libraries/Firmata" />
     <antcall target="unzip">
       <param name="archive_file" value="Temboo-1.1.2.zip" />
       <param name="archive_url" value="https://github.com/temboo/Temboo/archive/1.1.2.zip" />
       <param name="final_folder" value="${target.path}/libraries/Temboo-1.1.2" />
       <param name="dest_folder" value="${target.path}/libraries" />
     </antcall>
-	<move file="${target.path}/libraries/Temboo-1.1.2" tofile="${target.path}/libraries/Temboo" />
+    <move file="${target.path}/libraries/Temboo-1.1.2" tofile="${target.path}/libraries/Temboo" />
   </target>
 
   <!-- copy hardware folder -->

